DEV_SETUP.md

```diff
@@ -167,28 +171,54 @@ Create the shared directory.  If you have not modified `SHARED_DRIVE_ROOT`, then
 Once you have completed the above steps, you can use Docker to build
 and run all of the service containers. There are detailed instructions
 for setting up Docker in the [docker folder](docker/README.md). But the
-following should cover the needs of most developers:
+following should cover the needs of most developers.
+
 
+1. Install docker packages.
+
+    ```sh
     $ sudo apt install docker.io
     $ pip install docker-compose
     $ sudo adduser $USER docker
+    ```
 
-Log in as yourself again, to activate membership of the "docker" group:
+1. Log in as yourself again to activate membership of the "docker" group and
+   re-activate your virtualenv.
 
+    ```sh
     $ su - $USER
+    $ workon cchq
+    ```
+
+1. Ensure the Docker service is running.
 
-Ensure the Docker service is running:
+    ```sh
+    $ sudo systemctl status docker
+    # if service is not running, execute:
+    $ sudo systemctl start docker
+    ```
 
-    $ sudo service docker status
+1. Ensure the elasticsearch config files are world-readable (their containers
+   will fail to start otherwise).
 
-Bring up the Docker containers for the services you probably need:
+    ```sh
+    chmod 0644 ./docker/files/elasticsearch*.yml
+    ```
 
-    $ scripts/docker up postgres couch redis elasticsearch zookeeper kafka minio
+1. Bring up the Docker containers.
 
-or, to detach and run in the background, use the `-d` option:
+    ```sh
+    $ ./scripts/docker up
+    # Or, use '-d' option to detach and run in the background:
+    $ ./scripts/docker up -d
+    ```
```

Comment:
How about put the `-d` option first and provide the comment to run in the foreground if you'd prefer, since most are going to run in the background? I know putting the slightly more complex command first is strange but I have to imagine it is the preferred one.